server/command.go

```diff
@@ -278,6 +283,55 @@ func executeHelp(p *Plugin, c *plugin.Context, header *model.CommandArgs, args .
 	return responsef(helpText)
 }
 
+func executeImport(p *Plugin, c *plugin.Context, header *model.CommandArgs, args ...string) *model.CommandResponse {
+	posts, err := p.API.GetPostsForChannel(header.ChannelId, 0, 1)
+	if err != nil {
+		return responsef(err.Error())
+	}
+
+	fileID := posts.ToSlice()[0].FileIds[0]
+
+	var file []byte
+	file, err = p.API.GetFile(fileID)
+	if err != nil {
+		return responsef(err.Error())
+	}
+
+	var info *model.FileInfo
+	info, err = p.API.GetFileInfo(fileID)
+	if err != nil {
+		return responsef(err.Error())
+	}
+
+	var config Config
+	configErr := json.Unmarshal(file, &config)
+	if configErr != nil {
+		return responsef(configErr.Error())
+	}
+
```

Comment:
0/5 maybe add an extra check that the result of the decode is not empty, effectively checking against invalid inputs that are valid JSON.